ert/data/record/_record.py

```diff
@@ -469,6 +473,7 @@ async def load_collection_from_file(
     mime: str,
     length: int = 1,
     is_directory: bool = False,
+    smry_keys: Optional[List[str]] = None,
```

Comment:
Was just wondering whether it would be possible to send directly `RecordTransformation` instead?

Reply:
It probably is, and do the branching one level up the call-stack. But I am not sure why.

Reply:
As discussed yesterday; due to the command line support, this function would still need to get all arguments instead of `RecordTransformations` themselves.